tools/gke/kubernetes_api.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python2.7
```

Comment:
I did not know about this until you mentioned!. I looked at it and noticed that it does not expose all functionality I need (for example, a service cannot accept multiple container port list: https://github.com/pjs7678/python-kubernetes/blob/master/kubernetes/service.py). Also, the package isn't actively being maintained (the last update was 9 months ago). 

Better to write our own stuff for now - it is simple enough.